src/wp-includes/block-template.php

```diff
@@ -358,3 +358,34 @@ function _resolve_template_for_new_post( $wp_query ) {
 		$wp_query->set( 'post_status', 'auto-draft' );
 	}
 }
+
+/**
+ * Register a template.
+ *
+ * @param string       $template_name  Template name in the form of `plugin_uri//template_name`.
+ * @param array|string $args           Object type or array of object types with which the taxonomy should be associated.
+ * @param array|string $args           {
+ *     @type string        $title                 Optional. Title of the template as it will be shown in the Site Editor
+ *                                                and other UI elements.
+ *     @type string        $description           Optional. Description of the template as it will be shown in the Site
+ *                                                Editor.
+ *     @type string        $content               Optional. Default content of the template that will be used when the
+ *                                                template is rendered or edited in the editor.
+ *     @type string[]      $post_types            Optional. Array of post types to which the template should be available.
+ *     @type string        $plugin                Uri of the plugin that registers the template.
```

Comment:
Reviewing the tests this looks optional.

```suggestion
 *     @type string        $plugin                Optional. Uri of the plugin that registers the template.
```

Also, is it the URI or the slug?

Reply:
Good catch! Done in b1f9ec20d3406b4060a52b39a86eded9f56aacb6.